### DISCUSSION TITLE
Airbyte Reverse Proxy 403 Forbidden Issue

### DISCUSSION THREAD

**Discussion by my-mai** [airbytehq/airbyte#45395] [Deployment]
Hello,

I'm trying to deploy Airbyte on an AWS EC2 Amazon Linux 2023. 

Was able to install docker and airbyte using abctl. Able to access, login to airbyte using ssh proxy. 

I'm attempting to set up a reverse proxy using nginx and certbot. I was able to reach the login screen using the domain but running into the following 403 error when trying to login using the credentials from the abctl local credentials command (these credentials work when I'm SSH into the server and access it from localhost:8000). 

error below: 

"Sorry, something went wrong.

You don't have the right permissions to take this action. (HTTP 403)"

stack trace: HttpError: errors.http.forbidden

Thanks

**Comment by benpoulson** [comment]
@my-mai I had the exact same problem, but I was able to get it working with the below nginx config.
I didn't investigate the exact root cause, but I figured it's happy with the additional proxy_set_header params that are set.

```
server {
    server_name airbyte.yourhostname.com;
    client_max_body_size 200M;  # Required for Airbyte API

    location / {
        proxy_pass http://127.0.0.1:8000;
        proxy_set_header Cookie $http_cookie;  # If you use Airbyte's basic auth
        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_read_timeout 3600;  # Adjust timeout as needed
    }

    listen 443 ssl http2;  # Managed by Certbot
    ssl_certificate /etc/letsencrypt/live/airbyte.yourhostname.com/fullchain.pem;  # Managed by Certbot
    ssl_certificate_key /etc/letsencrypt/live/airbyte.yourhostname.com/privkey.pem;  # Managed by Certbot
    include /etc/letsencrypt/options-ssl-nginx.conf;  # Managed by Certbot
    ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem;  # Managed by Certbot
}

server {
    listen 80;
    server_name airbyte.yourhostname.com;

    return 301 https://$host$request_uri;  # Redirect HTTP to HTTPS
}
```

[a comment by an automated account is not shown]
